app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const {} = require("./controllers/articles.controller");
```

Comment:
Lets only write code driven by our tests, currently we don't need any of these lines of code with empty objects so please remove :)